src/main/java/com/querydsl/webhooks/GithubReviewWindow.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2015 The Querydsl Team.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.querydsl.webhooks;
+
+import static java.time.ZonedDateTime.now;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.ScheduledFuture;
+
+import org.kohsuke.github.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.env.Environment;
+import org.springframework.scheduling.TaskScheduler;
+import org.springframework.scheduling.concurrent.ConcurrentTaskScheduler;
+
+import com.github.shredder121.gh_event_api.GHEventApiServer;
+import com.github.shredder121.gh_event_api.handler.pull_request.*;
+import com.github.shredder121.gh_event_api.model.Commit;
+import com.github.shredder121.gh_event_api.model.PullRequest;
+import com.google.common.base.Throwables;
+import com.google.common.collect.Maps;
+
+/**
+ * GitHub Review Window - A GitHub Webhook Implementation.
+ *
+ * <p />
+ * When using Pull Requests, it's often necessary to review their contents.
+ *
+ * <p />
+ * Out of the box this piece of software assumes a review window of 2 days.
+ *
+ * <p />
+ * usage:
+ *   {@code java [-Dduration.(labelName)=(durationString)] -jar gh-review-window-{version}-full.jar }
```

Comment:
Maybe it would be better to put the example for running the project in a `README.md`?

I normally don't put the `@author` line in code I write, but feel free to do it. It's just a matter of preference.